README.md

```diff
@@ -107,6 +107,22 @@ function formatter(response) {
 procore.get({base: '/me'}, { formatter })
 ```
 
+### Handling Multiple Procore Zones (MPZ)
+
+You can also add custom headers to the request. A typical use case would be to add support for
+[Multiple Procore Zones (MPZ)](https://developers.procore.com/documentation/mpz-headers)
+
+```javascript
+const procore = client(authorizer);
+// Create your own headers
+const headers = {
+  'Procore-Company-Id': procoreCompanyId,
+};
+
+// Pass the headers configuration
+procore.get({base: '/me'}, { headers })
```

Comment:
The `/me` endpoint is one of the few endpoints that does not require the Procore-Company-Id.

See `https://developers.procore.com/documentation/tutorial-mpz` 

```suggestion
procore.get({base: '/projects', qs: {company_id: ###}, { headers })
```